Platform/AlderlakeBoardPkg/Include/PlatformBoardId.h

```diff
@@ -25,8 +25,8 @@ Defines Platform BoardIds
 #define BoardIdAdlPSDdr5Rvp                           0x0B
 #define PLATFORM_ID_ADL_PS_DDR5_RVP                   0x0B
 
-#define BoardIdAdlPSDdr5Crb                           0x0D // MiniITX is CRB
-#define PLATFORM_ID_ADL_PS_DDR5_CRB                   0x0D
+#define BoardIdAdlPSDdr5Crb                           0x0C // MiniITX is CRB
```

Comment:
what's the value read from SMBUS for PS board? Often this value would not change once we know the value, not sure  why changes BoardIdAdlPSDdr5Crb here.

Reply:
the value read from SMBUS is 0xD but due to the offset workaround in line 125 in EcSupport.c
*BoardId = (SmbusInputPort1Info.InputPort1Fields.BoardId - ADL_SMBUS_BOARDID_OFFSET);
The reason with this workaround is we seeing same conflict (same BoardID) between ADLN CRB and ADLPS RVP
https://github.com/slimbootloader/slimbootloader/commit/0dc5b1da43db0232098b15ecb55a090903b874ee

Reply:
Updated the patch as per discussion